tutormfe/__about__.py

```diff
@@ -1,2 +1 @@
-__version__ = "14.0.2"
-
```

Comment:
You're right: this and the one in setup.py should be reverted for nightly.  We can reintroduce them at the proper time later.

Reply:
Yes, I believe you should leave the version number at `14.0.2`. It will be bumped to `15.x` when olive is merged back into master, and then master merged into nightly.